shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/presentation/FeedingsScreenUi.kt

```diff
@@ -116,14 +116,15 @@ private fun LoadingState() {
 }
 
 @Composable
-private fun EmptyState() {
+private fun EmptyState(isPendingTab: Boolean) {
     Box(
-        modifier = Modifier.fillMaxSize()
+        modifier = Modifier.fillMaxSize(),
```

Comment:
Can we make `fillMaxSize()` a part of `AnimealPopUpScreen` to avoid redundant wrappers? Or at least add a modifier parameter?